plugins/backstage-plugin-coder/docs/README.md

```diff
@@ -1,11 +1,18 @@
 # Plugin API Reference – Coder for Backstage
 
-For users who need more information about how to extend and modify the Coder plugin. For general setup, please see our main [README](../README.md).
+The Coder plugin for Backstage does follow semantic versioning.
```

Comment:
I think this is an important sentence, but should not necessarily be the first sentence of this doc. Can we change it to something that explains what the reader will learn from reading this doc? e.g.

> Learn how to build custom Backstage functions and components that use the Coder API